Add Chi_c1(3872) in the PDG database

vkucera opened the pull request.

vkucera committed:
Add HF particles in the PDG database

ddobrigk commented:
Many thanks, @vkucera ! I think this looks very reasonable. Are these three particles all that is currently being used as new or revised in terms of settings from the HF side? If so, then we can hopefully move on soon and then do the second step of removing the `TDatabasePDG` from `RecoDecay.h` (more info [here](https://alice.its.cern.ch/jira/browse/O2-3782))

vkucera committed:
Remove particles that should be fixed in ROOT

Masses have been updates directly in the ROOT database.
PR: https://github.com/root-project/root/pull/12968

vkucera commented:
@sawenzel @benedikt-voelkel Ready for merge.
